lib/_stream_writable.js

```diff
@@ -148,7 +150,7 @@ function WritableState(options, stream, isDuplex) {
 
   // Storage for data passed to the afterWrite() callback in case of
   // synchronous _write() completion.
-  this.afterWriteTickInfo = null;
+  this[kAfterWriteTickInfo] = null;
```

Comment:
Sorry, I should have been clearer. This change is good, but the `afterWriteTickInfo` property should still be retained as a non-enumerable alias.

e.g.

```js
Object.defineProperty(this, 'afterWriteTickInfo', {
  enumerable: false,
  get() { return this[kAfterWriteTickInfo]; },
  set(val) { this[kAfterWriteTickInfo] = val; }
});
```

The alias can then be separated deprecated.

Reply:
I believe `Object.defineProperty()` is what was used originally and calling that at every invocation is what caused the significant performance regression. Might be able to add it to `WritableState`'s prototype once on initial module load though.

Reply:
Actually, this property has only existed for two months (since 2205f85b2caadee425a0a86bd8cc3bcb889e4bfe). I doubt anyone has written any code that relies on this internal state property, so an alias probably isn't even necessary...

Reply:
@jasnell  What is the purpose of defining a symbol property and then adding an extra allias for it that accesses the original property through a getter/setter? Wouldn't it destroy performance and add redundant complexity?

BTW, I'm not sure if this was discussed before, but... why don't we switch to private class fields. They are much faster than public attributes and they are fully hidden from user.

Reply:
@mscdex good points! if the property has gone out in a release however, removing it still makes it semver-major unfortunately. Adding the alias to the prototype would work.

@Hakerh400 ... using the symbol avoids the performance hit, adding the alias avoids it needing to be a semver-major change.

Reply:
Ok, added the alias